docs/api/ReactWrapper/reduce.md

```diff
@@ -41,8 +41,8 @@ class Foo extends React.Component {
 
 ```jsx
 const wrapper = mount(<Foo />);
-const total = wrapper.find(Bar).reduce((amount, n) => amount + n.prop('amount'));
-expect(total).to.equal(16);
+const total = wrapper.find(Bar).reduce((amount, n) => amount ? amount + n.prop('amount') : n.prop('amount'));
```

Comment:
actually, on the first iteration `amount` is the first item - that's the behavior of `reduce` when omitting the initial value.